app/js/sendToChart.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * function to generate the data object for the chart displayed on the front end.
+ * @returns the data object for graph.js
+ */
+async function sendToChart() { 
+    let users = await createUsersObject();
```

Comment:
As Christian said, I'm sure this data is already available in the app, should be possible to take it from elsewhere.

Reply:
I've hooked updateChart into updateScoreTable now, so the same fetch request is being used to populate both the table and the chart